.agents/manifest.md

```
@@ -0,0 +1,62 @@
+<!--
+Source: Based on Obsidian community guidelines
+Last synced: See sync-status.json for authoritative sync dates
+Update frequency: Check Obsidian releases repo for validation requirements
+-->
+
+# Manifest rules (`manifest.json`)
+
+### Required Fields
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)